script.js

```diff
@@ -0,0 +1,33 @@
+
+//properties added to the object: ‘input’ and ‘output’ as an array
+
+var chatbot = {
+    input: ["hello", "how are you", "how is your day", "what do you do"],
+    output: [ "hi", "greaatt!", "good", "software developer" ]
+ };
+
+//print variable
+//console.log(chatbot);
+
+//created a function called ‘reply’.
+function reply() {
+    let question = document.getElementById("input").value.toLowerCase();
+
+//conditional statement
+    if (chatbot.input[0] === question) {
+        document.getElementById("output").value = chatbot.output[0]
+    } else if (chatbot.input[1] === question) {
+        document.getElementById("output").value = chatbot.output[1]
+    }else if (chatbot.input[2] === question) {
+        document.getElementById("output").value = chatbot.output[2]
+    }else if (chatbot.input[3] === question) {
+        document.getElementById("output").value = chatbot.output[3]
+    } else {
+        document.getElementById("output").value = "Enter a valid value"
+    }
+    
+}
+
+
+//attach a 'click' event listener to the <button> element defined in the HTML file.
+document.getElementById("submit").addEventListener("click", function() {reply()})
```

Comment:
For your callback function, it's not necessary to create an anonymous function (a function with no name, i.e. `function() { ... }`) and call `reply()` inside of it. It works, but you can just pass in the name of your `reply` function directly. Like this:

```javascript
 document.querySelector("#submit").addEventListener("click", reply);
```

Or your callback function can be an anonymous function like below. If you do this, you don't need to write a separate `reply` function because you have included it inline:
```javascript
 document.querySelector("#submit").addEventListener("click", function() {
   const question = document.querySelector("#input");
   if(question.value === "hello"){
       document.querySelector("#output").textContent = "hi";
   } else{
       document.querySelector("#output").textContent = "I don't understand that command. Please enter another.";
   }
});
```